source/application/PGM/PGMAppController.h

```diff
@@ -58,6 +70,13 @@ class PGMAppController : public CLSAppController
 	virtual void createDetectorPanes();
 	/// create pane for the scan configuration views
 	virtual void createScanConfigurationPanes();
+
+	/// Returns true if the list of regions of interest contains the given ROI.
+	bool containsRegionOfInterest(QList<AMRegionOfInterest*> roiList, AMRegionOfInterest *regionOfInterest) const;
```

Comment:
We can live with this now though. and an issue for code refactor will be good too.